Sources/BSON/BSONNulls.swift

```diff
@@ -116,6 +136,16 @@ internal struct BSONMinKey: BSONValue, Equatable {
         self = BSONMinKey()
     }
 
+    /// Converts this `BSONMinKey` to a corresponding `JSON` in relaxed extendedJSON format.
+    internal func toRelaxedExtendedJSON() -> JSON {
+        self.toCanonicalExtendedJSON()
+    }
+
+    /// Converts this `BSONMinKey` to a corresponding `JSON` in canonical extendedJSON format.
+    internal func toCanonicalExtendedJSON() -> JSON {
+        ["$minKey": .number(1)]
```

Comment:
ditto using the literal here and elsewhere

Reply:
done